e2e/testcases/helm_sync_test.go

```diff
@@ -197,7 +197,7 @@ service:
 			  ],
 			  "wordpressEmail": "test-user@example.com"
 			},
-			"valuesFileSources": [
+			"valuesFileRefs": [
 			  {
 				"kind": "ConfigMap",
```

Comment:
the kind should have a hard-coded default, so it doesn't need to be in the test, but maybe that depends on a future PR to change the default behavior.

Reply:
We actually don't have the kind anymore at all as a field so this is just a remnant of me being rushed and mistakenly leaving this here. I'll take it out.